USAGE.md

```diff
@@ -64,7 +64,42 @@ layer description, view configuration.
 Saved layer configuration files can be opened in the ATT&CK Navigator to restore a layer you've worked on previously. To
```

Comment:
This is surely a merge conflict, but the table of contents in the help page is... pretty wild. For some reason it has the ".narrow" class which makes the width 20px:

<img width="780" alt="image" src="https://user-images.githubusercontent.com/28868888/127546870-af0c55ed-65c4-4b76-a487-b0bd20edfc52.png">